variables.tf

```diff
@@ -1,3 +1,18 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# These variables must be set when using this module.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "region" {
+  description = "The AWS region this bucket should reside in."
+  type        = string
+}
+
```

Comment:
why would you do this? the default is always the one set in the provider... the one of the callee which is totally fine., no need to require it...

Reply:
true, I"ve rolled back the change